geonode_mapstore_client/client/js/selectors/resource.js

```diff
@@ -105,9 +105,10 @@ export const canEditPermissions = (state) => {
     const groups = compactPermissions.groups || [];
     const organizations = compactPermissions.organizations || [];
     const user = state?.security?.user;
+    const userGroups = user?.info.groups.filter(group => group !== 'anonymous') || [];
```

Comment:
@DavidQuartz here you're doing assumptions that the client shouldn't need to do. You want to get all the (django) groups the user is a member of, and just check if ANY of these groups have the `manage` permission. Filtering the `anonymous` group shouldn't be required, moreover, it couples the rule to information that the client shouldn't rely on. If this filtering is required it means that there's something wrong with the following logic.